test/unit/item.test.js

```diff
@@ -193,6 +193,40 @@ describe('Item', function () {
         });
     });
 
+    describe('.getPath()', function () {
+        it('should return correct path for 2 level nested item', function () {
+            const collection = new sdk.Collection(fixtures.nestedCollectionV2),
+                f1 = collection.items.members[0],
+                r1 = f1.items.members[0];
+
+            expect(r1.getPath()).to.deep.equal(['multi-level-folders-v2', 'F1', 'F1.R1']);
+        });
+
+        it('should return correct path for 1 level nested item', function () {
+            const collection = new sdk.Collection(fixtures.nestedCollectionV2),
+                r1 = collection.items.members[2];
+
+            expect(r1.getPath()).to.deep.equal(['multi-level-folders-v2', 'R1']);
+        });
+
+        it('should return correct path for 3 level nested item', function () {
+            const collection = new sdk.Collection(fixtures.nestedCollectionV2),
+                f2 = collection.items.members[1],
+                f3 = f2.items.members[0],
+                r1 = f3.items.members[0];
```

Comment:
Unused variables.

Reply:
They aren't unused, next declaration is using previous.  Instead if I access request directly from collection it would be one long ugly chained access.

Reply:
Check if `oneDeep` helps here?